Celeste.Mod.mm/Patches/AudioState.cs

```diff
@@ -1,36 +1,37 @@
-﻿#pragma warning disable CS0626 // Method, operator, or accessor is marked external and has no attributes on it
+#pragma warning disable CS0626 // Method, operator, or accessor is marked external and has no attributes on it
 
 using MonoMod;
 
 namespace Celeste {
     class patch_AudioState : AudioState {
 
-        public float AmbienceVolume;
+        public float? AmbienceVolume;
 
         [MonoModConstructor]
         [MonoModReplace]
         public void ctor() {
-            AmbienceVolume = 1f;
+            AmbienceVolume = null;
```

Comment:
like you mentioned in the desc, these initializations aren't needed anymore since nullables default to null. not much point in having three unnecessary patches